euler_support/urdf/euler_macro.xacro

```diff
@@ -159,13 +164,13 @@
     <!--euler to motoman link-->
     <joint name="${prefix}platform_to_arm" type="fixed">
       <parent link="${prefix}base_link"/>
-      <child link="${prefix}motoman_base_link"/>
+      <child link="${prefix+robot_prefix}base_link"/>
       <origin xyz="0.067518 0 2.016" rpy="3.1415 0 0"/>
       <axis xyz="0 0 0"/>
     </joint>
     <!--motoman to end effector link-->
-    <joint name="${prefix}motoman_link_t_to_eef_base" type="fixed">
-      <parent link="${prefix}motoman_link_t"/>
+    <joint name="${prefix+robot_prefix}link_t_to_eef_base" type="fixed">
+      <parent link="${prefix+robot_prefix}link_t"/>
       <child link="${prefix}eef_eef_base"/>
```

Comment:
I know you didn't add this but the end-effector name has a repeated "eef" term in it.  Could you fix that?